Diz.Test/CartTitleTests.cs

```diff
@@ -0,0 +1,121 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Xml;
+using Diz.Core;
+using Diz.Core.model;
+using Diz.Core.serialization.xml_serializer;
+using Diz.Core.util;
+using ExtendedXmlSerializer;
+using FluentAssertions;
+using IX.StandardExtensions;
+using Xunit;
+
+namespace Diz.Test
+{
+    public static class CartNameTests
+    {
+        // Bytes for a Cart Name from a SNES header
+        // "Marvelous - Mouhitotsu no Takara-jima (Japan).sfc"
+        // StartOffset(h): 00007FC0, EndOffset(h): 00007FD4, Length(h): 00000015 
+        // 21 bytes
+        private static byte[] RawRomBytes => new byte[] {
+            0xCF, 0xB0, 0xB3, 0xDE, 0xAA, 0xD7, 0xBD, 0x20, 0x20, 0x20, 
+            0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 
+            0x20
+        };
+        private static string ExpectedTitleStr => "ﾏｰｳﾞｪﾗｽ".PadRight(RomUtil.LengthOfTitleName);
```

Comment:
padright works ok here because these are all 1 byte chars (I think).  this wont work for 2 byte chars, test that out.